machine-learning/01-medulloblastoma_data_prep.Rmd

```diff
@@ -243,7 +246,8 @@ agg_exprs_df %>%
   dplyr::filter(SYMBOL == dup_symbol)
 ```
 
-Is this consistent with what we expect?
+Is this consistent with what we expect?  
+Note that the values in each set of parentheses below represent the gene expression value of each CALM3 duplicate for that sample.
```

Comment:
```suggestion
Note that the values in each set of parentheses below represent the gene expression value of each *CALM3* duplicate for that sample.
```
I could be wrong, but I think generally when we refer to the expression of a gene it needs to italicized? I always get it mixed up though, so double check me on this.

Reply:
You're correct!